openstack/containerorchestration/v1/bays/results.go

```diff
@@ -0,0 +1,42 @@
+package bays
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// Extract is a function that accepts a result and extracts a bay resource.
+func (r commonResult) Extract() (*Bay, error) {
+	var bay *Bay
```

Comment:
What does `s` stand for in this function? I thought it was the deserialized bay, so I'm a bit confused about changing the variable from bay to s.